Code/JavaWrappers/gmwrapper/src-test/org/RDKit/PicklingTests.java

```diff
@@ -134,6 +134,25 @@ public void testIssue220() {
 		}
 	}
 
+	@Test
+	public void testDepicklerCrash() {
+		// this was github #2466
+		String smi = "CC";
+		ROMol m1 = RWMol.MolFromSmiles(smi);
+		m1.getAtomWithIdx(0).setAtomMapNum(-1,false);
+		m1.getAtomWithIdx(1).setAtomMapNum(-1,false);
+		
+		Int_Vect pickle = m1.ToBinary();
```

Comment:
I’m a bit confused.  We shouldn’t be able to make a pickle that can’t be unpickled.

Reply:
ah, good point. I was using this test before I fixed the bug that caused the crash.
I'll pull this test back out.